modules/arm_plugin/demos/coco_detection_android_demo/README.md

```diff
@@ -0,0 +1,130 @@
+# Coco Detection Android Demo
+
+![Running result](https://user-images.githubusercontent.com/47499836/188678748-a63f6010-d165-4372-927f-106b4a4e7c06.png)
+
+This is a demo for ARM processors Android devices. Using object detection model to reach the coco datasets' infomation. We use pre-trained models from [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo): [ssdlite_mobilenet_v2](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/ssdlite_mobilenet_v2) for object detection in coco dataset, [efficientdet-d0-tf](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/efficientdet-d0-tf), [pelee-coco](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/pelee-coco).
+
+The application reads frames from your device's camera or emulator's camera, and processes the network to detect the coco objects' locations and attributes. Then draw it on the image.
+
+The current openvino engine does not currently support models in INT8 MIXed format for reasoning on ARM devices, but will support models in this format in the near future and will achieve better performance.
+
+| Model | FP16 | FP32 |
+| --- | --- | --- |
+| ssdlite_mobilenet_v2 | 120ms | 120ms |
+| efficientdet-d0-tf | 800ms | 800ms |
+| pelee-coco | 220ms | X |
```

Comment:
Any performance benchmark is a quite sensitive point. It requires an info about hardware/software setup at least. So we recommend not to provide the numbers.

Reply:
Thanks, I will remove it.